### DISCUSSION TITLE
[Support]: 0.14.0-beta1 - unable to view log page in app

### DISCUSSION THREAD

**Discussion by ChrisChapman** [blakeblackshear/frigate#11617] [General Support]
### Describe the problem you are having

From the UI I am unable to load any of the log pages with the error below:

### Version

0.14.0-beta1

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
[2024-05-29 15:16:18] frigate.api.app                ERROR   : Exception on /logs/frigate [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/opt/frigate/frigate/api/app.py", line 487, in logs
    dateEnd = cleanLine.index("  ")
ValueError: substring not found
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

n/a

**Comment by ChrisChapman** [comment]
To add, the call from the app to the API is: http://xxxxx/api/logs/frigate?start=-100

**Reply by NickM-27** [reply]
we will need to see the contents of that api call

**Reply by ChrisChapman** [reply]
Sure:

curl "http://xxx/api/logs/frigate?start=-100" ^
  -H "Accept: */*" ^
  -H "Accept-Language: en-US,en;q=0.9" ^
  -H "Connection: keep-alive" ^
  -H "Cookie: frigate_token=xxx" ^
  -H "Referer: http://xxx/logs" ^
  -H "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" ^
  -H "X-CACHE-BYPASS: 1" ^
  -H "X-CSRF-TOKEN: 1" ^
  --insecure


Thanks!

**Reply by NickM-27** [reply]
sorry, I mea the contents of the response

**Reply by ChrisChapman** [reply]
It's just a generic Nginx 500 "Internal Server Errrr" - had to dig through the frigate log to get the ` dateEnd = cleanLine.index("  ")`  line
